### PR TITLE
Implement `device enableDeviceUrl/disableDeviceUrl/hasDeviceUrl/getDeviceUrl`

### LINKED ISSUE #383: There is access to "deviceURL" configurations from resin-cli

### PR DESCRIPTION
**Pull request by lifeeth**
Closes #383

### CODE REVIEW

File: lib/actions/device.coffee [hunk 1]
```diff
@@ -186,6 +186,70 @@ exports.reboot =
 		resin = require('resin-sdk')
 		resin.models.device.reboot(params.uuid).nodeify(done)
 
+exports.enableDeviceUrl =
+	signature: 'device enableDeviceUrl <uuid>'
```
Comment (jviotti) [inline lib/actions/device.coffee]: Maybe we can use more human-like command names to be consistent with the other commands? Like `device enable url <uuid>`? Notice you can freely use multi-words commands.

Also, the device in `deviceUrl` is probably redundant with the initial `device` command namespace?

Comment (dfunckt) [inline lib/actions/device.coffee]: If we do change the command name, I think it's important to convey it's _public_. In the Dashboard and [docs](http://docs.resin.io/management/devices/#enable-public-device-url) we consistently call the feature "Public Device URL", which could probably map to `device public-url enable` and `device public-url disable` nicely (I'm unfamiliar with CLI's naming conventions but I think you get the picture).

Comment (lifeeth) [inline lib/actions/device.coffee]: That sounds good - Let me do that.

**Comment by jviotti** [comment]
Looks great, I'll merge in a second. @lifeeth Do you need me to publish a new version ASAP, or can this wait to be chunked with other changes?

**Comment by lifeeth** [comment]
@jviotti the sooner the better as we have a use for enabling the public url in testing.

**Comment by jviotti** [comment]
@lifeeth I've published v4.3.0 to npm, which includes the device public url commands.

**Pull request merged**